elements/document/nuxeo-document-create.js

```diff
@@ -200,6 +200,7 @@ Polymer({
                   class="docTypeButton"
                   on-tap="_selectType"
                   data-args$="[[type]]"
+                  disabled$="[[!_canCreate()]]"
```

Comment:
_canCreate has two dependencies, `canCreate` and `creating`:
```
  _canCreate() {
    return this.canCreate && !this.creating;
  },
```
This means that we'd like it to be recomputed when they changed:
```suggestion
                  disabled$="[[!_canCreate(canCreate, creating)]]"
```

Reply:
Yes I agree, ;) I did it on my local, `disabled$="[[!_canCreate(canCreate)]]"` but I loosed it before I push, I think when I did the screenshot. Thanks